libethereum/ChainParams.cpp

```diff
@@ -98,7 +133,8 @@ ChainParams ChainParams::loadConfig(
 	json_spirit::read_string_or_throw(_json, val);
 	js::mObject obj = val.get_obj();
 
-	validateFieldNames(obj, c_knownChainConfigFields);
+    validateConfigJson(obj);
+    validateFieldNames(obj, c_knownChainConfigFields);
```

Comment:
Did you mean to leave both of these validations here?
So now we have two separate list of allowed fields to maintain?

Reply:
the other one could be removed from this place. once we come to a standart in genesis config.
the question remains is wether we should use constants instead of strings in scheme validation functions.

Reply:
I don't think we should validate it two times. So let's not merge it then until we have final version